gp-includes/router.php

```diff
@@ -28,7 +28,7 @@ public function request_uri() {
 	}
 
 	public function request_method() {
-		return gp_array_get( $_SERVER, 'REQUEST_METHOD', 'GET' );
+		return wp_unslash( gp_array_get( $_SERVER, 'REQUEST_METHOD', 'GET' ) );
```

Comment:
I don't think that `$_SERVER['REQUEST_METHOD']` will ever include quotes/slashes.